stable_baselines3/common/logger.py

```diff
@@ -21,7 +27,7 @@ class KVWriter(object):
     Key Value writer
     """
 
-    def writekvs(self, kvs: Dict) -> None:
+    def writekvs(self, kvs: Dict, step: int) -> None:
```

Comment:
please give it a default value, so we keep the previous behavior if not using tb